template.yaml

```diff
@@ -277,6 +277,30 @@ Resources:
               ArnLike:
                 kms:EncryptionContext:aws:logs:arn: !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:*
 
+  RpPublicKeyCacheTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: RpPublicKeyCache
+      AttributeDefinitions:
+        - AttributeName: clientId
+          AttributeType: S
+        - AttributeName: keyId
+          AttributeType: S
+      KeySchema:
+        - AttributeName: clientId
+          KeyType: HASH
+        - AttributeName: keyId
+          KeyType: RANGE
+      BillingMode: PAY_PER_REQUEST
+      SSESpecification:
+        SSEEnabled: true
+        KMSMasterKeyId: !GetAtt MainKmsKey.Arn
+      PointInTimeRecoverySpecification:
+        PointInTimeRecoveryEnabled: true
```

Comment:
I know you've done this to appease checkov, but I think we should remove this and make an exception. It would be actively harmful to restore from a historical backup of this data, and it is intended to be fetched if not present anyway.